boards/riscv/tlsr9518adk80d/tlsr9518adk80d.dts

```diff
@@ -7,3 +7,41 @@
 /dts-v1/;
 
 #include "tlsr9518adk80d-common.dtsi"
+
+/ {
+	led_pool{
+		compatible = "gpio-leds";
+
+		out {
+			gpios = <&gpiob 6 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	pwm_pool {
+		compatible = "pwm-leds";
+		out {
+			pwms = <&pwm0 2 PWM_MSEC(20) PWM_POLARITY_NORMAL>,
```

Comment:
PWM LED at index 0 it's "indication" (yellow if exists otherwise blue)
PWM LEDs at indexes 1..3 RGB fixture according to LES's color, if blue is used just RG
Looking at schematic current data seems to be wrong...
Also for other boards should be checked.

Reply:
inspetc all of them